pkg/controller/baremetalhost/host_state_machine.go

```diff
@@ -42,6 +42,9 @@ func (hsm *hostStateMachine) handlers() map[metal3v1alpha1.ProvisioningState]sta
 		metal3v1alpha1.StateExternallyProvisioned: hsm.handleExternallyProvisioned,
 		metal3v1alpha1.StateMatchProfile:          hsm.handleMatchProfile,
 		metal3v1alpha1.StateReady:                 hsm.handleReady,
+		metal3v1alpha1.StateManualCleaning:        hsm.handleManualCleaning,
```

Comment:
If you are going to add new states, please update the [state machine documentation](https://github.com/metal3-io/baremetal-operator/blob/master/docs/baremetalhost-states.md).